gui/src/system/SystemPages.tsx

```diff
@@ -374,19 +386,31 @@ async function submitConfig(edited: boolean) {
       null : JSON.parse(mappingvalvenormcloseds[i].value.toLowerCase())
   }
   console.log(entries);
-  const success = await sendConfig(serverIp() as string, {id: configName, mappings: entries} as Config) as object;
+  const response = await sendConfig(serverIp() as string, {id: configName, mappings: entries} as Config) as Response;
+  const success = response as object;
   const statusCode = success['status' as keyof typeof success];
```

Comment:
I know that this snippet wasn't yours, but this is totally invalid and exploits the TS type system incorrectly.